config/config.go

```diff
@@ -252,6 +252,27 @@ func (c *Config) extend(extensionConfig Config) {
 			c.OrderedRules = append(c.OrderedRules, ruleID)
 		} else {
 			// Rule exists, merge our changes into the base.
+			if currentRule.Description != "" {
+				baseRule.Description = currentRule.Description
+			}
+			if currentRule.Entropy != 0 {
+				baseRule.Entropy = currentRule.Entropy
+			}
+			if currentRule.SecretGroup != 0 {
+				baseRule.SecretGroup = currentRule.SecretGroup
+			}
+			if currentRule.Regex != nil {
+				baseRule.Regex = currentRule.Regex
+			}
+			if currentRule.Path != nil {
+				baseRule.Path = currentRule.Path
+			}
+			if len(currentRule.Tags) > 0 {
+				baseRule.Tags = currentRule.Tags
```

Comment:
@9999years Should this append to the base rule rather than overwrite? Same goes with Keywords?

Reply:
Sure, that's fine.

Reply:
Overwrite does give more control. #1384 points out that the docs explicitly say rules are overridden.